### ISSUE TITLE
[native] Naming convention clarification

### ISSUE THREAD

**Issue by Mionsz** [prestodb/presto#18721]
I am not sure if the valid and 'should use' name for presto-native-execution was declared. This makes some people confused and not sure if or not the listed bellow are same product:
- Prestissimo
- PrestoCpp
- presto-native-execution
- Presto native server

Could we have this clarified and replaced in the documentation or can we make a vocabulary that will clarify this?
@majetideepak

**Comment by rohanpednekar** [comment]
@psbell-meta Could you please take this further?

**Comment by mbasmanova** [comment]
CC: @aditi-pandit @amitkdutta @xiaoxmeng

**Comment by Mionsz** [comment]
@spershin The purpose of my question is more to clarify the naming convention - as I had an occasion to see it can confuse new users - and also it was a kind of management open question to ask what should be used like in internal documentation etc. Btw. having documentation refactored would also be a nice-to-do.

Nevertheless a simple vocabulary in documentation should be created additionally to clarify what is the proper name and how people use to call this query engine. I am not sure but this can have an impact on google search ranges and project visibility - it easier to find product if the name is clarified :).

Btw If there is some kind of consensus I can do documentation refactoring.

**Comment by spershin** [comment]
@Mionsz 

> Btw If there is some kind of consensus I can do documentation refactoring.

That is probably the biggest issue.
Everyone is working on it, but no one really thinks that one unified name is important enough to get it sorted out or takes initiative to actually do something about it.

I would go as far here as to suggest to use "Presto Native", unless some in context it does not fit. :)

**Comment by Mionsz** [comment]
@mbasmanova @spershin Could you rise this issue or discuss it on corporate/team level and give a kind of permanent answer or is the "Presto Native" the one? I ask as I do feel "Presto Native" is not a catchy name at all, "Prestissimo" is lots better and will probably give more consistent search results than two words name and this implicates far greater 'ranges'.

**Comment by Mionsz** [comment]
Then we could write: "Prestissimo is a Presto Worker native execution query engine written using low-level programming languages like C++ and overcoming all of JAVA performance bottlenecks by utilizing CPU vector operations instructions sets..."

**Comment by majetideepak** [comment]
I concur with @mbasmanova  and @Mionsz. We should use Prestissimo as the main name. Just like Databricks calls their stack Photon.
@Mionsz do you still have the bandwidth to implement this name change?

**Comment by Mionsz** [comment]
@majetideepak Yes I can handle this but probably not faster than in 2 weeks from now .

**Comment by majetideepak** [comment]
@Mionsz that works! I will open an issue for this meanwhile.
